server/index.js

```diff
@@ -1,11 +1,25 @@
+const mongoose = require('mongoose'); // mongoose is a module for mongodb, it allows us to talk to mongoDb
+mongoose.connect('mongodb://localhost/fetcher');

```

Comment:
It may be more appropriate to make the database connection in the db directory, then import it to the server.

Reply:
yeah , i will clean the code up